tyk-pro/templates/deployment-mdcb.yaml

```diff
@@ -115,6 +120,10 @@ spec:
           periodSeconds: 10
           timeoutSeconds: 3
           failureThreshold: 3
+      {{- with .Values.mdcb.image.pullSecrets }}
+      imagePullSecrets:
+      {{- toYaml . | nindent 12 }}
```

Comment:
While this change is a really good catch, not sure there are still 12 units to indent (gotta love yaml!)

Reply:
Thanks @excieve. I'm all tested here and think we're good to go. Resolving both